src/main/java/org/opentripplanner/graph_builder/GraphBuilder.java

```diff
@@ -241,6 +247,22 @@ public static GraphBuilder create(
                     )
             );
         }
+
+        if (DataOverlay.isOn()) {
```

Comment:
```suggestion
        if (OTPFeature.DataOverlay.isOn()) {
```
The purpose of the OTPFeature is to immediately see that I (as a person who don´t now what a DataOverlay is) can skip reading the code block. So, it help that the `OTPFeature` is used and that it. is not statically imported. In other places using static imports may help the readability, but not here.